### DISCUSSION TITLE
[Feature request] Changing the font size

### DISCUSSION THREAD

**Discussion by vitaliylag** [google/mozc#811] [Ideas]
The font size of suggestions and other Japanese text is too small. It would be great to add an option to choose the font size.

**Comment by yukawa** [comment]
Before jumping into a solution, can we discuss a bit more on
 * what's the common way to adjust font size (or logical DPI) with the attached screens in your platform?
 * how `mozc_renderer` can access to the system level font settings programmatically?

As far as I know, the system level DPI setting is already taken into account at least in Windows so I guess you are using Mozc in some other platform, right? If so, do you happen to know how `mozc_renderer` can access to the system level font settings (or DPI settings) programmatically?
https://github.com/google/mozc/blob/f79e092ac7068f6bbdacb2b069641771e7cb1f43/src/renderer/renderer_style_handler.cc#L206-L225

**Comment by yutayu** [comment]
I think that Font size is small , too.

**Comment by tsomeq** [comment]
I want to change font family also.
Some character has very similar shape for other one, like `1` (one) and `l` ( small letter of L ).
I do not need UI dialog for change it. Editing the preference file directly may be acceptable.
